upb/decode.h

```diff
@@ -7,10 +7,14 @@
 
 #include "upb/msg.h"
 
-UPB_BEGIN_EXTERN_C
```

Comment:
Isn't this macro more convenient?

Reply:
The macro is a little bit nice, but clang-format doesn't know about it so it formats it incorrectly. Since the benefit it was providing is small, it didn't seem worth keeping.

I suppose clang-format might format it correctly if it looked more like a function, eg

```
UPB_BEGIN_EXTERN_C();
```

But overall I am trying to move away from things that are too clever.